python/lsst/resources/tests.py

```diff
@@ -10,7 +10,10 @@
 # license that can be found in the LICENSE file.
 from __future__ import annotations
 
-__all__ = ["GenericTestCase", "GenericReadWriteTestCase"]
+__all__ = [
```

Comment:
This reformatting seems odd. I think what happened is you added a new function to the end of the list, that triggered black to reformat it, then you removed the new function but black sees no reason to change it back to how it was. To reduce git history churn can you please take it back to how it was and make sure to squash the commit history so the change never happens?